interactions/api/gateway/client.py

```diff
@@ -111,7 +113,10 @@ def __init__(
             "timeout": 60,
             "autoclose": False,
             "compress": 0,
+            "proxy": str(proxy) if isinstance(proxy, ProxyConfig) else None,
+            "proxy_auth": proxy.auth if isinstance(proxy, ProxyConfig) else None,
         }
+        print(self._options)
```

Comment:
No

Reply:
ah yes i should've searched for my debug print statements, huh